presto-cli/src/main/java/com/facebook/presto/cli/Console.java

```diff
@@ -368,7 +368,15 @@ private static boolean process(QueryRunner queryRunner, String sql, OutputFormat
 
     private static MemoryHistory getHistory()
     {
-        return getHistory(new File(getUserHome(), ".presto_history"));
+        String prestoCliHistoryFilePath = System.getenv("PRESTO_CLI_HISTORY_FILE");
```

Comment:
All the other CLI environment variable are just `PRESTO_`, so let's make this `PRESTO_HISTORY_FILE`.